core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -131,6 +131,7 @@ registry.ci.openshift.org/origin/4.10:jenkins quay.io/openshift/origin-jenkins:4
 registry.ci.openshift.org/origin/4.10:jenkins-agent-base quay.io/openshift/origin-jenkins-agent-base:4.10 quay.io/openshift/origin-jenkins-agent-base:4.10.0 quay.io/openshift/origin-jenkins-agent-base:latest
 registry.ci.openshift.org/origin/4.10:jenkins-agent-maven quay.io/openshift/origin-jenkins-agent-maven:4.10 quay.io/openshift/origin-jenkins-agent-maven:4.10.0 quay.io/openshift/origin-jenkins-agent-maven:latest
 registry.ci.openshift.org/origin/4.10:jenkins-agent-nodejs quay.io/openshift/origin-jenkins-agent-nodejs:4.10 quay.io/openshift/origin-jenkins-agent-nodejs:4.10.0 quay.io/openshift/origin-jenkins-agent-nodejs:latest
+registry.ci.openshift.org/origin/4.10:jenkins-sync-plugin-scripts quay.io/openshift/origin-jenkins-sync-plugin-scripts:4.10 quay.io/openshift/origin-jenkins-sync-plugin-scripts:4.10.0 quay.io/openshift/origin-jenkins-sync-plugin-scripts:latest
```

Comment:
Is this "sync-plugin" for "jerkins"?  Or "Jenkins scripts that implement a sync plugin" for "Jenkins"?  Just want to understand the name so I can piece together a naming scheme.

Reply:
> Is this "sync-plugin" for "jerkins"? Or "Jenkins scripts that implement a sync plugin" for "Jenkins"? Just want to understand the name so I can piece together a naming scheme.

yes the code we are trying to test is the sync-plugin for jenkins ... the old https://github.com/openshift/jenkins-sync-plugin that dev ex took over from the fabric guys back in the early 3.x days a few years ago.

That jenkins plugin gets included in the set of jenkins plugings included in the openshift jenkins image at https://github.com/openshift/jenkins

I now have the golang based e2e's for it in that actual repo .... PR tests in that repo use those tests.  I'd like for the PR tests in https://github.com/openshift/jenkins to use those tests as well.  This image is suppose to contain the code for those tests in its content.

The "scripts" element comes from my attempt to work off naming conventions @adambkaplan followed with https://github.com/openshift/release/pull/20806 that he got approval from you on @smarterclayton 

Of course if my naming attempts are not ideal, please advise.

OK hopefully that additional context provides what you need @smarterclayton  ... thanks

Reply:
@gabemontero why do we need to mirror the scripts image to quay?

My understanding is that the "scripts" image will contain our sync plugin e2e tests which get invoked by `go test` or equivalent. The promotion stanzas instruct CI to make the image available via the `stable` imagestream. We on

Mirroring to quay is a separate thing that is customer facing. This image should only be used in our CI environment - I don't think we need to push it out to end users.

Reply:
IIRC @adambkaplan (it has been a while since I've gotten attention on this) the change stemmed from running 

`make openshift-image-mirror-mappings `

likewise IIRC I needed to do this after adding https://github.com/openshift/release/pull/21474/files#diff-d039dd1d4471aed28d6c59497dba358abeeb9611c6dd122de1d60acf6d659a37R21 otherwise some of the verification failed

but again, it has been a while

I'll drop that second comment, update the PR, and see what sorts out

if it is not needed, apologies @smarterclayton for the unneeded churn

Reply:
ok I have removed the `make openshift-image-mirror-mappings` commit and this file is no longer updated @adambkaplan @smarterclayton

Reply:
Image mirror mapping jobs shouldn't fail here - it seems that the make command logic is trying to create a mirror mapping that we don't want.

This didn't happen in my prior PR for csi-driver-shared-resource: https://github.com/openshift/release/pull/20967

@openshift/test-platform why does `make openshift-image-mirror-mappings` want to add a mirrored image here?

Reply:
in the interim @adambkaplan I've updated this PR for 
1) only update master branch ... assuming CI will handle mirroring to 4.10 once that master / 4.10 split occurs
2) changed the name from scripts to e2es since that is what we are dealing with here

let's see how the test jobs here react

Reply:
> Image mirror mapping jobs shouldn't fail here - it seems that the make command logic is trying to create a mirror mapping that we don't want.
> 
> This didn't happen in my prior PR for csi-driver-shared-resource: #20967
> 
> @openshift/test-platform why does `make openshift-image-mirror-mappings` want to add a mirrored image here?

hey @adambkaplan went back and revisited this, and I see from your csi-driver-shared-resource work https://github.com/openshift/release/blob/1f41c71e615c06be295abfd2d6e88cbfa995dd32/core-services/image-mirroring/openshift/mapping_origin_4_10#L89 

`registry.ci.openshift.org/origin/4.10:csi-driver-shared-resource-scripts quay.io/openshift/origin-csi-driver-shared-resource-scripts:4.10 quay.io/openshift/origin-csi-driver-shared-resource-scripts:4.10.0 quay.io/openshift/origin-csi-driver-shared-resource-scripts:latest`

is this not an indication that your changes either *DID* or since your PR merge now *DO* in fact require a mirrored image

and we should move forward with merging this update for e2e tests for jenkins-sync-plugin with a mirrored image change (that has to be approved by Clayton).